pkgs/by-name/op/openscad-unstable/package.nix

```diff
@@ -135,12 +133,25 @@ clangStdenv.mkDerivation rec {
   # tests rely on sysprof which is not available on darwin
   doCheck = !stdenv.hostPlatform.isDarwin;
 
+  # remove unused submodules, to ensure correct dependency usage
+  postUnpack = ''
+    ( cd $sourceRoot
+      for m in submodules/OpenCSG submodules/mimalloc submodules/Clipper2
+      do rm -r $m
+      done )
+  '';
+
   postInstall = lib.optionalString stdenv.hostPlatform.isDarwin ''
     mkdir $out/Applications
     mv $out/bin/*.app $out/Applications
     rmdir $out/bin
   '';
 
+  preCheck = ''
```

Comment:
what is this used for?

Reply:
This fixed a couple of tests, trying to access `$HOME` ..

Reply:
Weird, can you point to some tests that were failing?

Reply:
I checked it again and didn't need it after all. I think it may have just been already failed tests, trying to write error logs into HOME ....